### PR TITLE
add ppg docs

### PR DESCRIPTION
**Pull request by nikolasburk**

### CODE REVIEW

File: content/100-getting-started/01-quickstart-prisma-postgres.mdx [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+title: 'Quickstart'
+metaTitle: 'Quickstart with TypeScript & Prisma Postgres'
+metaDescription: 'Get started with Prisma ORM in 5 minutes. You will learn how to send queries to a SQLite database in a plain TypeScript script using Prisma Client.'
+search: true
+dbSwitcher: ['sqlite', 'prisma-postgres']
+slugSwitch: /getting-started/quickstart-`
+sidebar_custom_props: { badge: '5 min' }
+---
+
+In this Quickstart guide, you'll learn how to get started from scratch with Prisma ORM and a remote **Prisma Postgres** database in a plain **TypeScript** project. It covers the following workflows:
```
Comment (petradonka) [inline content/100-getting-started/01-quickstart-prisma-postgres.mdx]: @nikolasburk Do you think "remote" is needed here? It somehow implies to me that there's a non-remote PPG as well.

File: content/100-getting-started/01-quickstart-prisma-postgres.mdx [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+- Connection pooling and caching (via [Prisma Accelerate](https://prisma.io/data-platform/accelerate))
+- Real-time database change events (via [Prisma Pulse](https://prisma.io/data-platform/pulse))
+
+## Prerequisites
+
+To successfully complete this tutorial, you need:
+- a [Prisma Data Platform](https://console.prisma.io/) (PDP) account
+- Node.js v16.13.0 or higher (learn more about [system requirements](/orm/reference/system-requirements))
+
+## 1. Set up a Prisma Postgres database in the PDP Console
```
Comment (petradonka) [inline content/100-getting-started/01-quickstart-prisma-postgres.mdx]:
```suggestion
## 1. Set up a Prisma Postgres database in the Platform Console
```

File: content/100-getting-started/01-quickstart-prisma-postgres.mdx [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+1. In the **Region** dropdown, select the region that's closest to your current location, e.g. **US East (N. Virginia)**.
+1. Click the **Create project** button.
+
+At this point, you'll be redirected to the **Dashboard** where you will need to wait for a few seconds while the status of your database changes from **`PROVISIONING`**, to **`ACTIVATING`** to **`CONNECTED`**.
+
+Once the green **`CONNECTED`** label appears, your database is ready to use!
+
+## 2. Download example and install dependencies
+
+Copy the `try-prisma` command that's shown in the Console UI, paste it into your terminal and execute it.
```
Comment (petradonka) [inline content/100-getting-started/01-quickstart-prisma-postgres.mdx]:
```suggestion
Copy the `try-prisma` command that's shown in the Console, paste it into your terminal and execute it.
```

File: content/100-getting-started/01-quickstart-prisma-postgres.mdx [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+  --template prisma-starter
+  --connection-string __YOUR_DATABASE_CONNECTION_STRING__
+  --name hello-prisma-postgres
+  --install npm
+  --path .
+```
+
+Your connection string that should replace the `__YOUR_DATABASE_CONNECTION_STRING__` placeholder looks similar to this: `prisma+postgres://accelerate.prisma-data.net/?api_key=ey...`
+
+Once the `try-prisma` command has terminaed, navigate into the project directory and install dependencies:
```
Comment (petradonka) [inline content/100-getting-started/01-quickstart-prisma-postgres.mdx]:
```suggestion
Once the `try-prisma` command has terminated, navigate into the project directory and install dependencies:
```

Maybe completed would be better, aside from the typo?

File: content/100-getting-started/01-quickstart-prisma-postgres.mdx [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+```
+The query took 300.5655280000001ms.
+```
+
+## 6. Observe real-time events in your database
+
+The  [`src/realtime.ts`](./src/realtime.ts) script contains a demo for receiving real-time change [events](/pulse/database-events) from your database. You can start the script as follows:
+
+```
+npm run realtime
```
Comment (petradonka) [inline content/100-getting-started/01-quickstart-prisma-postgres.mdx]: Unrelated to docs, I remember running this, and not realizing that it's only subscribed to the User table. Maybe we can make that more clear in the terminal output, if it's still the case that it's not crystal clear?

File: content/100-getting-started/01-quickstart-prisma-postgres.mdx (hunk 1 above)
Comment (nikolasburk) [reply content/100-getting-started/01-quickstart-prisma-postgres.mdx]: good catch, I dropped "remote" here 👍

File: content/100-getting-started/index.mdx [hunk 6]
```diff
@@ -40,14 +41,27 @@ Explore our products that make it easy to build and scale data-driven applicatio
 
 [**<SignalStream />Prisma Pulse**](/pulse) allows you to react to database changes with type-safe model streams in real-time.
 
-</BorderBox>
+[**<PrismaPostgres />Prisma Postgres**](/orm/overview/databases/prisma-postgres) is a managed PostgreSQL service that gives you an _always-on_ database pay-as-you-go pricing. Prisma Postgres is currently in [Early Access](/platform/maturity-levels#early-access).
```
Comment (petradonka) [inline content/100-getting-started/index.mdx]:
```suggestion
[**<PrismaPostgres />Prisma Postgres**](/orm/overview/databases/prisma-postgres) is a managed PostgreSQL service that gives you an _always-on_ database with pay-as-you-go pricing. Prisma Postgres is currently in [Early Access](/platform/maturity-levels#early-access).
```

Comment (petradonka) [reply content/100-getting-started/index.mdx]: @sneub might have an idea also on the 1 sentence "what is PPG" messaging. Maybe we need some pre-made options in the value messaging doc? E.g. in 5 words, in 1 sentence, in 3 sentences, in a paragraph. Then we can keep reusing that.

File: content/100-getting-started/01-quickstart-prisma-postgres.mdx [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+
+For reference, this is what the command looks like (note that the `__YOUR_DATABASE_CONNECTION_STRING__` placeholder must be replaced with _your_ actual database connection string):
+
+```
+npx try-prisma@latest
+  --template prisma-starter
+  --connection-string __YOUR_DATABASE_CONNECTION_STRING__
+  --name hello-prisma-postgres
+  --install npm
+  --path .
```
Comment (petradonka) [inline content/100-getting-started/01-quickstart-prisma-postgres.mdx]:
```suggestion
npx try-prisma@latest \
  --template prisma-starter \
  --connection-string __YOUR_DATABASE_CONNECTION_STRING__ \
  --name hello-prisma-postgres \
  --install npm \
  --path .
```

@nikolasburk, I tried this, and we need some newline escapes. I also got an error for now for --connection-string:

```
❯ npx try-prisma@latest \
  --template prisma-starter \
  --connection-string __YOUR_DATABASE_CONNECTION_STRING__ \
  --name hello-prisma-postgres \
  --install npm \
  --path .
Cannot run command, you made some mistakes:

Unknown flag "connectionString"
X Unknown flag "path"
```

Comment (jharrell) [reply content/100-getting-started/01-quickstart-prisma-postgres.mdx]: Should be `database-url` and `path` is now removed.
```suggestion
npx try-prisma@latest \
  --template prisma-starter \
  --database-url __YOUR_DATABASE_CONNECTION_STRING__ \
  --name hello-prisma-postgres \
  --install npm
```

**Review by mhessdev — APPROVED** [review]

**Pull request merged**